README.md

```diff
@@ -522,37 +563,60 @@
     if (test)
       return false;
 
+    // acceptable (only for very simple statements and early function return)
+    if (test) return false;
+
     // good
     if (test) {
       return false;
     }
 
-    // good
-    function () { return false; }
+    // bad
```

Comment:
It's not the curly braces, but the lack of the new line.  Using without a curly braces doesn't work with functions.